### PR TITLE
Fix agi bug -- allow commas in money fields.

### PR DESCRIPTION
**Pull request by bytheway875**

### CODE REVIEW

**Comment by tgrathwell** [comment]
ugh what an embarrassing bug

The stripping of the commas at type cast time is good, but is the decimal migration adding anything here? Are we sure the migration updates existing values successfully?

I expect AGI is "supposed" to be a whole dollar amount but I guess this would handle the case where someone enters 1234.00; though currently it just drops the fractional part which also seems fine

**Comment by tgrathwell** [comment]
the `MoneyType` was my attempt to carry forward knowledge from GCF but somehow I completely missed the comma part which now makes me feel quite the fool

Screen Shot 2021-08-24 at 11 54 52 AM

**Comment by bytheway875** [comment]
bleh, i was supposed to have reverted the change to decimal but i dont think i did.

File: spec/features/ctc/complete_intake_spec.rb [hunk 1]
```diff
@@ -33,13 +33,15 @@ def strip_html_tags(text)
     expect(page).to have_selector("h1", text: I18n.t('views.ctc.questions.filed2020.title'))
     click_on I18n.t('general.negative')
     expect(page).to have_selector("h1", text: I18n.t('views.ctc.questions.filed2019.title'))
-    choose I18n.t('views.ctc.questions.filed2019.did_not_file')
+    choose I18n.t('views.ctc.questions.filed2019.filed_full')
```
Comment (tgrathwell) [inline spec/features/ctc/complete_intake_spec.rb:36]: there's a separate "client who has filed in 2019" scenario in this file that would be a better place to put this, so we have one flow for a non-filer and another test that does the AGI for the primary and spouse

Comment (bytheway875) [reply spec/features/ctc/complete_intake_spec.rb]: done!

**Pull request merged**